pkg/config/config.go

```diff
@@ -20,6 +20,8 @@ type BtrfsOptionsConfig struct {
 type OverlayOptionsConfig struct {
 	// IgnoreChownErrors is a flag for whether chown errors should be
 	// ignored when building an image.
+	// This can take any value accepted by Go's https://pkg.go.dev/strconv#ParseBool
+	// Search the source code for the term "string bool" to find other places which behave similarly.
```

Comment:
For all three comments in this file… Referring to `ParseBool` is a correct description of the current readers. But, well, do we _want_ to commit to accepting things like `[01tTfF]`? (Think possible Rust consumers of the config.)

I’d rather document just `"true"`/`"false"`, and treat the behavior of `drivers/overlay`, accepting the other values, as an undocumented… (non?)bug when processing invalid input.

Especially when, right now, the `docs/containers-storage.conf.5.md` man page does not document the values _at all_, adding the `ParseBool` references here could be construed as a commitment to support the values in the config file. I’d rather not do that.

---

(Ceterum autem censeo, the `c/storage/drivers` subpackage should almost entirely become private; and the option parsing should pass these Go structures down to `drivers/overlay` directly instead of round-tripping through `[]string`. But also, I’m scared of the c/storage option parsing / loading code.)